app/bundles/PageBundle/Form/Type/RedirectListType.php

```diff
@@ -12,15 +11,19 @@
  */
 class RedirectListType extends AbstractType
 {
-    public function __construct(
-        private CoreParametersHelper $coreParametersHelper
-    ) {
+    public function __construct()
```

Comment:
Can we remove this construct if it's not doing anything?

Reply:
Removed. Thanks!